vms/platformvm/vm_test.go

```diff
@@ -413,56 +413,6 @@ func TestInvalidAddValidatorCommit(t *testing.T) {
 	require.ErrorIs(reason, txexecutor.ErrTimestampNotBeforeStartTime)
 }
 
-// Reject attempt to add validator to primary network
-func TestAddValidatorReject(t *testing.T) {
```

Comment:
This test broke since we can't meter `AddValidatorTx`. We could rewrite this to go through the new tx type, but it wasn't clear to me what this was actually testing.